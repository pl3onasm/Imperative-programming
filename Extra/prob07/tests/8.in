45615684061561035046465321321213213648546854084651021023041560454605456040500152648608464165843434034085640684896465165465465466974878789465486465045653564156
1456222356888612385423585858523231000005454568852636966
