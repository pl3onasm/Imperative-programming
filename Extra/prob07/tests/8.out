31324669509280316083279711018151125826946483436848443177977231012177501921586332291325484955396204713849
618973105187214010009791135177494064211237682544022022
